lib/std/os/bits/darwin.zig

```diff
@@ -23,13 +23,6 @@ pub const sockaddr = extern struct {
     family: sa_family_t,
     data: [14]u8,
 };
-pub const sockaddr_storage = extern struct {
```

Comment:
Why are these removed?

Reply:
They were removed in favor of `std.x.os.Socket.Address.Native.Storage` which is the same definition as `sockaddr_storage`, though defined just once with platform differences taken into consideration. Though, I can revert the change as mentioned in the comment about `msghdr`.